index.js

```diff
@@ -1437,11 +1341,6 @@ ContainerWindow.prototype = {
     }
   },
 
-  _shutdownPlusButtonMenu() {
```

Comment:
This caused `TypeError: this._shutdownPlusButtonMenu is not a function` during a 2.3.0 -> 2.4.0 upgrade on Fx 56.